topics/cluster-spec.md

```diff
@@ -513,6 +523,11 @@ set the slots to their normal state again. The same command is usually
 sent to all other nodes to avoid waiting for the natural
 propagation of the new configuration across the cluster.
 
+Starting from Valkey 8.0, Valkey clusters introduce the ability to elect a primary in empty shards.
+This behavior ensures that even when a shard is in the process of receiving its first slot,
+a primary can be elected. This prevents scenarios where there would be no primary available in the
+empty shard to handle redirected requests from the official slot owner,
+thereby maintaining availability during the live reconfiguration.
 ### ASK redirection
```

Comment:
A blank line before the next heading never hurts and it's safer for correct rendering. (The rendering on the website could be different to how github renders it.)